studio/src/rte/rte-field.js

```diff
@@ -232,11 +287,21 @@ class RteField extends LitElement {
                     'data-perpetual': { default: null },
                     'data-promotion-code': { default: null },
                     'data-wcs-osi': { default: null },
+                    'data-template': { default: null },
                     title: { default: null },
                     target: { default: null },
                     'data-analytics-id': { default: null },
+                    text: { default: '' },
```

Comment:
@Axelcureno in my version, I didn't need to add the text attribute. Since it is already it is a child node of type text. I would keep it this way.

Reply:
True, I have removed text attribute.